src/main/java/io/xpring/xrpl/model/XRPTransaction.java

```diff
@@ -115,15 +124,47 @@
   XRPPayment paymentFields();
 
   /**
-   * Constructs an {@link XRPTransaction} from a {@link Transaction}.
+   * The timestamp of the transaction reported in Unix time (seconds).
+   *
+   * @see "https://xrpl.org/basic-data-types.html#specifying-time"
+   */
+  @Nullable
+  Integer timestamp();
+
+  /**
+   * (Omitted for non-Payment transactions) The Currency Amount actually received by the Destination account.
+   * Use this field to determine how much was delivered, regardless of whether the transaction is a partial payment.
+   *
+   * @see "https://xrpl.org/transaction-metadata.html#delivered_amount"
+   */
+  @Nullable
+  String deliveredAmount();
```

Comment:
I would recommend typing this as `Optional<String>` instead of making it `@Nullable`